tools/sstableloader/test/unit/org/apache/cassandra/tools/LoaderOptionsTest.java

```diff
@@ -215,5 +215,40 @@ public void checkPermission(Permission perm, Object context)
             System.setSecurityManager(null);
         }
     }
+
+    // Copied from OfflineToolUtils
```

Comment:
Ultimately ended up doing this instead of having sstableloader-test depend on test classes, as the test classes already depend on the sstableloader build, this didn't feel like it was worth doing, so just went for copying the minimal amount of shared code.